pages/data-providers/[data-provider-id]/dois.css

```diff
@@ -0,0 +1,74 @@
+%item {
+  flex: 3 3 90%;
+  margin: 0 var(--component-spacer-x, 1rem) var(--component-spacer-y, 1.5rem);
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-start;
+  align-items: flex-start;
+  padding: var(--component-padding-y, 1rem) 0;
+
+  & > * {
+    @extend %item;
+  }
+}
+
+.card-one {
+  flex: 2 2 60%;
+  width: 100%;
+}
+
+.card-two {
+  flex: 1 1 30%;
+  width: 100%;
+}
```

Comment:
Still do not know what to put there. The second one should stay being 'export'. The first one is a mystery at this moment 🙂

Reply:
Could you please remind me if we already decided? Or is it still a mystery?

Reply:
Yes, it is. I've not been working on this yet, sorry.